lib/project_types/script/layers/infrastructure/script_project_repository.rb

```diff
@@ -56,7 +57,8 @@ def get
             script_name: script_name,
             extension_point_type: extension_point_type,
             language: language,
-            script_config: script_config_repository.get!
+            script_config: script_config_repository.get!,
+            input_query: read_input_query,
```

Comment:
Theres a couple other `Domain::ScriptProject.new(...)` calls in this file. Should we add `input_query` to them? I'm noticing it isn't very DRY right now though

Reply:
Revisiting this now. I didn't add it to all of them initially because there wouldn't be one when creating the script, at least not now. But on second thought, if there were one, we'd wanna populate the field. And we will eventually have one when creating a script.

I extracted a `build_script_project`. The tests failed when requiring a `script_config` in `ScriptProjectRepository#create`, so I left it `nil` as it were before in that code path.